btx/interfaces/psana_interface.py

```diff
@@ -14,9 +14,9 @@ def __init__(self, exp, run, det_type,
         self.det_type = det_type # detector name, str
         self.track_timestamps = track_timestamps # bool, keep event info
         self.seconds, self.nanoseconds, self.fiducials = [], [], []
-        self.event_receiver = event_receiver
-        self.event_code = event_code
-        self.event_logic = event_logic
+        self.event_receiver = event_receiver # 'evr0' or 'evr1', str
+        self.event_code = event_code # event code, int
+        self.event_logic = event_logic #
```

Comment:
Sorry, one final request: maybe we could add `self.event_code = event_code # if True, retain events with event_code; if False, keep all other events` or something similar?